### ISSUE TITLE
Dependency update required (deprecated graceful-fs required by fstream 0.1)

### ISSUE THREAD

**Issue by yoher** [richardbolt/node-archiver#1]
Hello Richard,

After installing `materialize-css`, I noticed a warning regarding `graceful-fs`:
`npm WARN deprecated graceful-fs@3.0.8: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible.`

This is caused by `node-archiver` requiring an old version of `fstream` requiring in turn the deprecated `graceful-fs` version: 

```
┬ materialize-css@0.97.6  
└┬ node-archiver@0.2.0  
 └┬ fstream@0.1.31  
  └ graceful-fs@3.0.8
```

`fstream` updated its `graceful-fs` dependency above 4.0.0 in release [1.0.9](https://github.com/npm/fstream/commit/53b0095290773e2e30fb1b768cdfe4db2a4baa5e).

Would it be possible to update the dependency in `node-archiver` to ensure it will stay compatible with future versions of Node.js?

Thank you - and please don't hesitate to get in touch if you need help for testing!
Yoann

**Comment by richardbolt** [comment]
Yes, that does need to be upgraded. I'll be able to do that shortly after the middle of this month.

Thanks for the heads up,

Richard

**Comment by richardbolt** [comment]
Dependencies for fstream and tar updated to latest and now published as 0.3.0 since no functionality changes were required to update these dependencies - https://www.npmjs.com/package/node-archiver

Thanks for the report!
